frontend/TestRun/Jenkins/SCTParameterWizard.svelte

```diff
@@ -0,0 +1,414 @@
+                    values: ["eu-west-1", "eu-west-2", "us-west-2", "us-east-1", "eu-north-1", "eu-central-1"],
+                },
+                azureRegion: {
+                    name: "Cloud Region",
+                    description: "Only supported regions are displayed",
+                    type: SelectParam,
+                    internalName: "azure_region_name",
+                    condition: (params, defs) => inBackendGroup(params.backend, "azure"),
+                    onShow: function (params) {
+                        if (!params[this.internalName]) {
+                            params[this.internalName] = this.values[0];
+                        }
+                    },
+                    onChange: function (e, params) {
+                        //empty
+                    },
+                    values: ["eastus"],
+                },
+                gceRegion: {
+                    name: "Cloud Region",
+                    description: "Only supported regions are displayed",
+                    type: SelectParam,
+                    internalName: "gce_datacenter",
+                    condition: (params, defs) => inBackendGroup(params.backend, "gce"),
+                    onShow: function (params) {
+                        if (!params[this.internalName]) {
+                            params[this.internalName] = this.values[0];
+                        }
+                    },
+                    onChange: function (e, params) {
+                        //empty
+                    },
+                    values: ["us-east1", "us-east4", "us-west1", "us-central1"],
+                },
+                availabilityZone: {
+                    name: "Availability Zone",
+                    description: "Availability zone for the region. Note that GCE only supports zones from a to d (with some exceptions)",
+                    type: SelectParam,
+                    internalName: "availability_zone",
+                    condition: (params, defs) => (true),
+                    onShow: function (params) {
+                        if (!params[this.internalName]) {
+                            params[this.internalName] = this.values[0];
+                        }
+                    },
+                    onChange: function (e, params) {
+                        //empty
+                    },
+                    values: Array.from({length: 8}, (_, idx) => String.fromCharCode("a".charCodeAt(0) + idx)),
+                },
+                provisionType: {
+                    name: "Provision Type",
+                    type: SelectParam,
+                    internalName: "provision_type",
+                    condition: (params, defs) => true,
+                    onChange: function (e, params) {
+                        //empty
+                    },
+                    values: ["spot", "on_demand", "spot_fleet"],
+                    labels: ["Spot Provisioning", "On Demand", "Spot Fleet"]
```

Comment:
I don't think we need to prettify the name in this case

Reply:
removed `Provisioning` from the first value